resources/styles/book-content/fun-in-physics.less

```diff
@@ -0,0 +1,9 @@
+.tutor-book-content {
```

Comment:
Is this defining a mixin named `tutor-book-content`? If so, can you include the parentheses to make it clear?